src/clusters.tsx

```diff
@@ -435,10 +445,14 @@ export class DaskClusterManager extends Widget {
     if (response.status !== 200) {
       const err = await response.json();
       void showErrorMessage('Cluster Start Error', err);
+      this.status = 'failed';
```

Comment:
Is there any way to set the status away from 'failed'? I'm not really understanding the semantics: does `ready` mean ready to start a cluster, or that a cluster is ready?

Reply:
Hmm. I guess it should be just two statuses, `ready` to start a cluster and `starting` all of my checks are just for whether status is starting. So I guess I could also make it a bool. Maybe that would be clearner. `starting: bool`

Reply:
I went with `isReady` so that the bool matches` isEnabled`.